src/studio/src/designer/frontend/packages/schema-editor/src/components/RestrictionField.tsx

```diff
@@ -89,6 +99,7 @@ export const RestrictionField = (props: IRestrictionFieldProps) => {
         className={classes.field}
         value={props.value}
         disableUnderline={true}
+        onKeyDown={onValueKeyDown}
```

Comment:
As you see 'key' can also refer to a keyboard 'key'... (not an issue here, just an example)